### ISSUE TITLE
Distance metric used in the DBSCAN 

### ISSUE THREAD

**Issue by vamshing** [gboeing/2014-summer-travels#1]
Hi,

I am just curious to know, what distance metric would the initial call for the clustering method would take. The documentation says that it would by default take the euclidean distance.

class sklearn.cluster.DBSCAN(eps=0.5, min_samples=5, metric='euclidean', algorithm='auto', leaf_size=30, p=None, random_state=None)[source]

Would that mean, the initial clusters are formed by the euclidean dist by taking lat/long data?

Thanks,
Vamshi

**Comment by gboeing** [comment]
Yes, the best bet for precision would be to either 1) project the point data first or 2) adjust the DBSCAN implementation to calculate great circle or vincenty distance
